src/pybind/mgr/dashboard/frontend/src/app/shared/api/cluster.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ClusterService {
+  baseURL = 'api/cluster';
+
+  constructor(private http: HttpClient) {}
+
+  getStatus(): Observable<string> {
+    return this.http.get<string>(`${this.baseURL}`, {
+      headers: { Accept: 'application/vnd.ceph.api.v0.1+json' }
+    });
+  }
+
+  updateStatus(status: string) {
+    return this.http.put(
+      `${this.baseURL}/${status}`,
```

Comment:
POST and PUT should always send the data to update in the body. The URL should only be used to identify the resource to be updated, but not to provide the specific data.

Reply:
Reverted.